aten/src/ATen/test/type_test.cpp

```diff
@@ -175,8 +175,8 @@ class OneForward(Interface):
 
 TEST(TypeEquality, TupleEquality) {
   // Tuples should be structurally typed
-  auto type = TupleType::create({IntType::get(), TensorType::get(), FloatType::get()});
-  auto type2 = TupleType::create({IntType::get(), TensorType::get(), FloatType::get()});
+  auto type = TupleType::create({IntType::get(), TensorType::get(), FloatType::get(), ComplexDoubleType::get()});
```

Comment:
Are these changes for testing `ComplexDoubleType::operator==`?

Reply:
yeah and also indirectly testing `ComplexDoubleType::get()`